stock_picking_report_custom_description/models/stock_rule.py

```diff
@@ -31,6 +31,14 @@ def _get_stock_move_values(
         )
         if values.get("sale_line_id"):
             line = self.env["sale.order.line"].browse(values["sale_line_id"])
-            res["description_picking"] = line.name
+            # Avoid printing double printing the name in the picking
+            description_picking = line.name.replace(
```

Comment:
Replacing unconditionally whole `display_name` seems too much, isn't it? Shouldn't it be only removed in case of having something more?

```python
if line != line.product_id.display_name:
```

Reply:
Well, not exactly, as you are doing an unconditional replace in any part of the text. Imagine a product called `Test` and this SOL line text:

```
Test
This is a Test
Another line
```

you will end up with

```
This is a Another Line
```

Reply:
Ok, I can improve it with a regex match :sweat_smile:

Reply:
Or simply an `startswith` plus a replace with just 1 repetition.

Reply:
I don't know if my suggestion is faster, but usually everything is faster than regex.